components/ErrorEventsDashboard.js

```diff
@@ -0,0 +1,142 @@
+import moment from 'moment'
+import { Component } from 'react'
+import { withAuth } from 'use-auth0-hooks'
+
+import { secureFetch } from '../util'
+
+class ErrorEventsDashboard extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      events: null,
+      eventsError: null
+    }
+    // TODO fix babel plugin so we can use class properties
+    // https://babeljs.io/docs/en/babel-plugin-proposal-class-properties
+    // https://nextjs.org/docs/advanced-features/customizing-babel-config
+    this.handleFetchErrors = this.handleFetchErrors.bind(this)
```

Comment:
Or you can define the method as:
```js
handleFetchErrors = async (force = false) => {...
```

Reply:
Yes, this has been fixed in the demo branch. I'll hold off on resolving for now because that branch contains commits which resolve this.